azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/mqtt/mqtt_transport.py

```diff
@@ -83,22 +109,22 @@ def __init__(self, auth_provider):
                 "dest": "disconnected",
             },
             {
-                "trigger": "_trig_send_event",
+                "trigger": "_trig_queue_action",
```

Comment:
Naming. I'm not clear from this what it does, exactly.

From what I can tell from the code, and our previous discussion, it seems that this enqueues an action, then potentially pops an action off the queue to resolve depending on the state.

Perhaps a better name would just be _trig_action?